app/helpers/discussions_helper.rb

```diff
@@ -107,8 +107,13 @@ def new_discussion_link(teaser_text, link_text)
     HTML
   end
 
-  def discussion_count_for_status(status, discussions)
-    discussions.scoped_query_by(discussion_filter_params, excluded_params: [:status], excluded_methods: [:page]).by_status(status).count
```

Comment:
I'm guessing this deletion would also allow deleting https://github.com/mumuki/mumuki-domain/blob/d9e9c64b50b183d9420f3a75d2c26cb31ea889be/app/models/concerns/with_discussion_status.rb#L7, as I don't see any other usages.

Though, understandably, we're not making a Domain PR just for that. Maybe we can remember to do it on master? Or not do it at all?

Reply:
I made the PR :sparkles:
(added a few things that were bothering me too while I was at it)